orttraining/orttraining/python/ort_trainer.py

```diff
@@ -660,16 +663,38 @@ def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_opti
         self._use_deterministic_compute = _use_deterministic_compute
         self.use_invertible_layernorm_grad = use_invertible_layernorm_grad
 
+        self._debug_conversion = _debug_conversion
+
         # use this special string to workaround a corner case that external loss_scale is passed into train_step as kwargs.
         # see prepare_input_and_fetches for more details.
         self.loss_scale_input_name = 'default_loss_scale_input_name'
 
         self._init_session()
 
+
+    def _debug_init_session(self):
+        self.no_postprocess_onnx_model = None
+        if not self._enable_internal_postprocess or not self._extra_postprocess:
+            self.no_postprocess_onnx_model = self.onnx_model_
```

Comment:
Isn't this a refernece instead of a copy? If self.onnx_model_ suffers post processing, wouldnt self.no_postprocess_onnx_model point to the post processed model too?

Reply:
I thought this might be copying the model because later when postprocessing, it does `self._onnx_model_ = postprocess.run_postprocess(self.onnx_model_)` . How would I explicitly copy by value?